ui/app/store.js

```diff
@@ -0,0 +1,8 @@
+import { createStore, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import reducer from './reducers';
+
+const middlewares = [thunk];
```

Comment:
You should consider adding a logging middleware here (redux-logger) if this is dev mode